CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,47 @@
+- Contributing to Nacto-care 🎓
+  Welcome to the EduWeb project! We're excited to have you contribute. Before you get started, please take a moment to review the guidelines for contributing.
+
+ - Table of Contents 📑
+    Getting Started
+       Code of Conduct
+    How to Contribute
+        Reporting Bugs
+        Suggesting Enhancements
+        Pull Requests
+        Coding Guidelines
+  - License
+
+- Getting Started 🚀
+- Code of Conduct 🤝
+ Please review and adhere to the Code of Conduct while participating in this project. Let's work together to create a positive and welcoming community.
+
+- How to Contribute
+- Reporting Bugs
+  If you encounter a bug or unexpected behavior, please open an issue and provide detailed information about the problem. Include steps to reproduce the issue, expected and actual results, and any relevant logs or screenshots.
+
+- Suggesting Enhancements
+  Have an idea for a new feature or improvement? Open an issue to suggest your enhancement. Describe the feature, its use cases, and how it could benefit the project.
+
+- Pull Requests 🔄
+  We welcome pull requests from contributors. To submit a pull request:
+
+1). Fork the repository and create a new branch.
+2).Make your changes and commit them with descriptive messages.
+3).Push your branch to your forked repository.
+4).Run npm run eslint in the root of the project to make sure your contributions are in line with our coding guidelines.
+5).Run npm run eslint-fix in the root of the project for ESLint to fix the issues it can.
+6).Open a pull request from your branch to the main repository.
+7).Provide a clear description of your changes and why they're valuable.
+  Please ensure your code adheres to the Coding Guidelines. Make sure to read PR Guidelines and Issue Guidelines first.
+
+- Coding Guidelines 📝
+   -Follow the established coding style and conventions.
+   -Write clear and concise code with meaningful comments when necessary.
+   -Test your changes thoroughly before submitting a pull request.
+   -Document new features, changes, or important concepts as needed.
+    
+- Contact Information
+For more ways to get in touch, refer to Harsil Jani[https://linkedin.com/in/harshil1]
```

Comment:
Please remove personal contact since moving forward, I will not be the only stakeholder on the project. Instead give a reference to nactoreorganization@gmail.com